routes/home.js

```diff
@@ -15,9 +15,10 @@ async function handleIndexRoute(req, res) {
     title: "Firefox Monitor",
     breach: breach,
   });
-}
+});
 
-router.get("/", handleIndexRoute);
-router.get("/monitor", handleIndexRoute);
```

Comment:
Was anything using this now absent “/monitor” endpoint?
Also, did you still want to add the GET /scans route redirecting to the index route so users won’t get 404s on refreshes or redirects?

Reply:
It's a legacy endpoint, I took the opportunity to remove it.